contrib/demo/clusters/kind/createKindClusters.sh

```diff
@@ -1,9 +1,34 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
+
+set -euxo pipefail
 
 DEMO_ROOT="$(dirname "${BASH_SOURCE}")/../.."
+TMPDIR=$(mktemp -d)
+
+clusters="$@"
+if [[ $# -eq 0 ]]; then
+  clusters="us-east1 us-west1"
+fi
+
+for name in ${clusters}; do
+  cat > ${TMPDIR}/${name}.config << EOF
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+name: ${name}
+networking:
+  apiServerAddress: "127.0.0.1"
+EOF
 
-kind create cluster --config ${DEMO_ROOT}/clusters/kind/us-east1.config --kubeconfig ${DEMO_ROOT}/clusters/kind/us-east1.kubeconfig
-sed -e 's/^/    /' ${DEMO_ROOT}/clusters/kind/us-east1.kubeconfig | cat ${DEMO_ROOT}/clusters/us-east1.yaml - > ${DEMO_ROOT}/clusters/kind/us-east1.yaml
+  kind delete cluster --name=${name} || true
+  kind create cluster --config ${TMPDIR}/${name}.config --kubeconfig ${TMPDIR}/${name}.kubeconfig
```

Comment:
```suggestion
  kind create cluster --config ${TMPDIR}/${name}.config --kubeconfig ${DEMO_ROOT}/clusters/kind/${name}.kubeconfig
```

Could we keep it like this?

And then add those .kubeconfigs to .gitignore ? 

Otherwise for some work I'm doing later it's not possible to fund the kubeconfigs and use them

You can always rely on `kind get kubeconfig --name $cluster` but, if you have other kind cluster in the system we can't know if they belong to this demo or something else.